tests/table/test_eval.py

```diff
@@ -367,6 +367,51 @@ def test_table_numeric_atan2_literals() -> None:
         )
 
 
+def test_table_numeric_arctanh() -> None:
+    table = MicroPartition.from_pydict({"a": [0.0, 0.5, 0.9, -0.9, -0.5, -0.0, math.nan]})
```

Comment:
Can you also test that for values `|x| >= 1`, the arctanh expression returns `Nan`?